### DISCUSSION TITLE
Is there a way to refresh library ?

### DISCUSSION THREAD

**Discussion by trexxeon** [jeffvli/feishin#121] [Q&A] [answered]
I can't seem to find anywhere to refresh the library, am I blind?

**Comment by jeffvli** [comment, marked as answer]
There's currently no way to perform a server refresh from Feishin.
I can add this in the next release.

**Reply by trexxeon** [reply]
Awesome, would be great to have that feature.

**Reply by marin6789** [reply]
Hey ! is this still possible to add ? i have to close the desktop app and reopen it again to see the last folders i have added on navidrome :(

**Comment by marin6789** [comment]
Hello, I would love to see this implemented.
It is very annoying to have to close the app and reopen it again to see newly added folders.
Thanks !

**Reply by jeffvli** [reply]
Added https://github.com/jeffvli/feishin/commit/13528eff098aed37fc4b9f8b14236383ccca6c0e

**Reply by marin6789** [reply]
Thanks ! Very sorry but i cannot see where it is
